### DISCUSSION TITLE
Sync button also syncs playback speed

### DISCUSSION THREAD

**Discussion by mherceg** [f1multiviewer/issue-tracker#240] [Feature requests]
Make the "Sync" button also change all the open windows' playback speed to the same value as the current one. 

I was watching the qualifying after the live session has already ended and wanted to watch portion of it on 1.5x, I had to go through all the open playback windows and change speed to keep them in sync.

**Comment by f1multiviewer** [comment]
Good idea! I wanted to do this before but I need to refactor a few things before this is possible, but working on this!

**Reply by f1multiviewer** [reply]
One of the things that is required for this is to also slow down live timing, including the track animations, which is *interesting* to say the least

**Reply by mherceg** [reply]
I would love to help if possible. Eagerly awaiting for the code to become public.
